### PR TITLE
added programming languages with 'G' and [...]

### PR DESCRIPTION
**Pull request by gabrielsozinho**
fix programming languages with 'S'

### CODE REVIEW

**Comment by gabrielsozinho** [comment]
@chilarai is this ok?

File: A-Z-dropdown.html [hunk 1]
```diff
@@ -42,6 +42,18 @@
...
+<option value="G">G</option>
+<option value="gamemakerlanguage">Game Maker Language</option>
+<option value="gamemonkeyscript">GameMonkey Script</option>
+<option value="gams">GAMS (General Algebraic Modeling System)</option>
+<option value="gap">GAP</option>
+<option value="gcode">G-code</option>
+<option value="gdscript">GDScript (Godot)</option>
+<option value="basic assembly language">Go</option>
+<option value="rpg">Grasshopper</option>
+<option value="inform">Groovy (Apache Groovy)</option>
```
Comment (chilarai) [inline A-Z-dropdown.html:55]: option value is different to name

File: A-Z-dropdown.html [hunk 2]
```diff
@@ -124,6 +136,7 @@
 <!-- End R -->
 
 <!-- Begin S -->
+<select name="select" id="select_S">
 <option value="s">S</option>
 <option value="s2">S2</option>
 <option value="s3">S3</option>
```
Comment (chilarai) [inline A-Z-dropdown.html:142]: why do we have dummy values here

**Review by chilarai — CHANGES_REQUESTED** [review]
Please fix these issues first

File: A-Z-dropdown.html (hunk 2 above)
Comment (gabrielsozinho) [reply A-Z-dropdown.html:142]: I didn't add it, I just corrected it as it was before. The languages ​​were with S (`option`s) but not with the `<select>` tag. Anyway, I'll fix it.

Comment (gabrielsozinho) [reply A-Z-dropdown.html:142]: @chilarai These are not dummy values.
image

Comment (chilarai) [reply A-Z-dropdown.html:142]: Thank you for that. Please resolve the conflicts and submit. Rest, I think its good to be merged

**Pull request merged**